src/main/java/com/example/todo/service/AccountServiceImpl.java

```diff
@@ -48,6 +49,6 @@ public AccountEntity getAccountByUsername(String username) {
     @Override
     public List<AccountEntity> getAllAccounts() {
 
-        return (List<AccountEntity>) userRepository.findAll();
+        return new ArrayList<>((Collection<? extends AccountEntity>) userRepository.findAll());
```

Comment:
Нет, такое решение не подойдет. Теперь, вместо каста к листу, будет каст к коллекции (хоть тип более общий, но тоже есть риск того, что вернется не объект, реализующий интерфейс `Collection`). Такой код скрывает проблему, а не решает её. Сделаем по-другому. `findAll()` возвращает `Iterable`, и тут лучше всего сработает следущее решение:
```suggestion
        List<AccountEntity> result = new ArrayList<>();
        userRepository.findAll().forEach(result::add);
        return result;
```